modules/services/skhd/default.nix

```diff
@@ -0,0 +1,49 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.skhd;
+
+in {
+
+  options = {
+    services.skhd.enable = mkOption {
+      type = types.bool;
+      default = false;
+      description = "Whether to enable the skhd hotkey daemon.";
+    };
+
+    services.skhd.package = mkOption {
+      type = types.package;
+      example = literalExample "pkgs.skhd";
```

Comment:
Are you planning to make a pr for nixpkgs or should I push my expression?

Reply:
I guess eventually all the credit goes to you, so I could only be the messenger. I can do that though.